### DISCUSSION TITLE
Large storage overage on Vercel Postgres

### DISCUSSION THREAD

**Discussion by e-hart** [vercel/community#2980] [Bug Report]
### Summary

I just looked at my usage panel and was surprised to see my storage size at 824 MB / 256 MB, waaay over quota.

This is a small, pre-launch, development DB with about 1000 rows to speak of. When I hover over the quota overage, a tip appears stating, "Total current project storage. 31MB of storage is from Postgres itself. [Learn more →](https://vercel.com/docs/storage/vercel-postgres/usage-and-pricing#storage)"

<img width="1277" alt="image" src="https://github.com/vercel/community/assets/97808888/35b575c5-ecfa-471f-85cd-41673b8614e1">

I'm not able to see anything in the Storage panel that might indicate what is wrong. I have 8MB of data written to the DB, and the panel itself says that the DB is only 31MB.

What could be causing this massive spike in storage that is pushing me over quota?

This is a Next.JS project interacting with the database via Prisma.  Everything looks and works fine, I'm just several times over quota....

### Steps to Reproduce

Create Postgres DB on Vercel Storage.

Use with Prisma.

**Comment by e-hart** [comment]
<img width="969" alt="image" src="https://github.com/vercel/community/assets/97808888/a68d3c5c-a81a-44aa-908f-67c2ec589f4f">

I have been watching my usage all month (as I've been trying to gauge what production usage would look like) and am entirely certain that until today have I not been over quota. It seems like this chart changed retroactively overnight....

**Comment by amyegan** [comment]
Thanks for reporting this. We had a display bug related to storage size reporting which is fixed now. You should no longer see this issue, so please let me know if you it is still happening for you 🙇
